app.js

```diff
@@ -15,6 +16,14 @@ app.get('/api/articles', getAllArticles)
 
 app.get('/api/articles/:article_id', getArticle)
 
+app.get('/api/articles/:article_id/comments', getArticleComments)
+
+app.use((err, req, res, next) => {
+    if(err.code === '22P02'){
+        res.status(400).send({msg: 'bad request'})
+    }
+    next(err)
+})
```

Comment:
Great use of a error handling middleware function catching PSQL errors
It would be even better if your error handling middleware functions were separated out to keep `app.js` tidy 🧹 
https://notes.northcoders.com/courses/js-back-end/error-handling#4-refactoring-to-individual-error-handling-blocks